content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,492 @@
+    - source_labels: [__address__]
+      target_label: __param_target
+    - source_labels: [__param_target]
+      target_label: instance
+    - target_label: __address__
+      replacement: localhost:9115  # The blackbox exporter’s real hostname:port. For Windows and macOS replace with - host.docker.internal:9115
+```
+
+Before applying the relabeling rules, the URI of a request Prometheus would make would look like this:
+`"http://prometheus.io/probe?module=http_2xx"`. After relabeling it will look like this `"localhost:9115/probe?target=http://prometheus.io&module=http_2xx"`.
+
+Now let us explore how each rule does that:
+
+First we take the values from the label `__address__` (which contain the values from `targets`) and write them to a new label `__param_target` which will add a parameter `target` to the Prometheus scrape requests:
+```yaml
+  relabel_configs:
+    - source_labels: [__address__]
+      target_label: __param_target
+```
+
+After this our imagined Prometheus request URI has now a target parameter: `"http://prometheus.io/probe?target=http://prometheus.io&module=http_2xx"`.
+
+Then we take the values from the label `__param_target` and create a label instance with the values.
+```yaml
+  relabel_configs:
+    - source_labels: [__param_target]
+      target_label: instance
+```
+Our request will not change, but the metrics that come back from our request will now bear a label `instance="http://prometheus.io"`.
+
+After that we write the value `localhost:9115` (the URI of our exporter) to the the label `__address__`. This will be used as the hostname and port for the Prometheus scrape requests. So that it queries the exporter and not the target URI directly. 
+```yaml
+  relabel_configs:
+    - target_label: __address__
+      replacement: localhost:9115  # The blackbox exporter’s real hostname:port. For Windows and macOS replace with - host.docker.internal:9115
+```
+
+Our request is now `"localhost:9115/probe?target=http://prometheus.io&module=http_2xx"`. This way we can have the actual targets there, get them as `instance` label values while letting Prometheus make a request against the blackbox exporter.
+
+Often people combine these with a specific service discovery. Check out the [configuration documentation](/docs/prometheus/latest/configuration/configuration) for more information. Using them is no problem, as these write into the `__address__` label just like `targets` defined under `static_configs`.
+
+That is it. Restart the Prometheus docker container and look at your [metrics](http://localhost:9090/graph?g0.range_input=30m&g0.stacked=0&g0.expr=probe_http_duration_seconds&g0.tab=0). Pay attention that you selected the period of time when the metrics were actually collected.
+
+# Summary
+
+In this guide, you learned how the multi-target exporter pattern works and how to run a blackbox exporter with a customised module and to configure Prometheus using relabeling to scrape metrics with prober labels.
```

Comment:
This is a bit run on, replace the first and with a comma.

Reply:
Will do.